src/KafkaFlow.Retry.API/Handlers/GetItemsHandler.cs

```diff
@@ -54,5 +55,21 @@ protected override async Task HandleRequestAsync(HttpRequest request, HttpRespon
                 await this.WriteResponseAsync(response, ex, (int)HttpStatusCode.InternalServerError).ConfigureAwait(false);
             }
         }
+
+        protected override string ResourcePath
```

Comment:
We could assign the ResourcePath on the constructor instead and avoid a re-calculation of the value of this property at each access to it. Do you think we can do that to this property and the other ones you changed similarly?

Reply:
I think it's better now. Thank you for the advice 👍

Reply:
Thank you!